pmlc/conversion/stripe_to_affine/convert_stripe_to_affine.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 Intel Corporation
+
+#pragma once
+
+#include <memory>
+
+namespace mlir {
+class FuncOp;
+class MLIRContext;
+template <typename T>
+class OpPassBase;
+using FunctionPassBase = OpPassBase<FuncOp>;
+class OwningRewritePatternList;
+
+/// Creates a pass to convert Stripe dialect to affine dialect.
+std::unique_ptr<FunctionPassBase> createConvertStripeToAffinePass();
+}  // namespace mlir
+
+namespace pmlc {
```

Comment:
Let's try to keep the namespace structure in-sync with the filesystem structure.

so: 
```
namespace pmlc {
namespace conversion {
namespace stripe_to_affine {
```

Reply:
Ok! For my understanding, do you strictly create a namespace per directory, even for directories with small files like this one?

Reply:
That's been the standard so far. We might not need a whole new directory for `stripe_to_affine` but we can always change it in the future.